random/.gitignore

```diff
@@ -0,0 +1 @@
+tmp/
```

Comment:
this is a test

Reply:
.... yep, that did .... if all of your comments become outdated, did your review ever exist?